ganttproject/src/net/sourceforge/ganttproject/client/RssUpdate.java

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2018 Oleksii Lapinskyi, BarD Software s.r.o
+
+This file is part of GanttProject, an opensource project management tool.
+
+GanttProject is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+GanttProject is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
+*/
+package net.sourceforge.ganttproject.client;
+
+public class RssUpdate {
+
+    private String version;
```

Comment:
Please do follow the [code style](https://docs.ganttproject.biz/development/#_hack)
Two spaces, `my` prefix (and `this` is not required). Also, these fields can be made final